dev-lang/mozart/mozart-2.0.1.ebuild

```diff
@@ -31,50 +32,47 @@ DEPEND="${RDEPEND}
 	test? ( dev-cpp/gtest:= )"
 
 S="${WORKDIR}/${PN}2-${PV}"
-BUILD_DIR="${S}/build"
-CMAKE_USE_DIR="${S}"
 
 src_prepare() {
 	if [[ -d "${WORKDIR}"/${PV} ]] ; then
 		eapply "${WORKDIR}"/${PV}
 	fi
-	eapply_user
+	default
 
-	touch "${S}"/stdlib/CMakeLists.txt
-	touch "${S}"/vm/vm/test/gtest/CMakeLists.txt
+	touch stdlib/CMakeLists.txt || die
+	touch vm/vm/test/gtest/CMakeLists.txt || die
 
-	cmake-utils_src_prepare
+	cmake_src_prepare
 }
 
 src_configure() {
-	mycmakeargs=(
+	local mycmakeargs=(
 		-DMOZART_BOOST_USE_STATIC_LIBS=OFF
-		-DEMACS=$(if use emacs; then echo /usr/bin/emacs; fi)
-		)
+		-DEMACS=$(usex emacs "/usr/bin/emacs" "")
```

Comment:
This is very suspicious, emacs USE flag without inheriting elisp-common. But of course it was like this before, so no blocker for this PR.

@keriharris

Reply:
This is being tracked as a separate issue in bug 194604 [1].

[1] https://bugs.gentoo.org/194604